modules/retail/pages/retail-formulas-intro.adoc

```diff
@@ -7,22 +7,38 @@ Formulas are pre-written Salt states, that are used to configure your {smr} inst
 You can use the {productname} {webui} to apply common {productname} formulas.
 For the most commonly used formulas, see xref:salt:formulas-intro.adoc[].
 
-Alternatively, you can use pre-written formulas as a starting point for your own custom formulas.
-Pre-written formulas are available from https://github.com/saltstack-formulas.
-For more information on custom formulas, see xref:salt:formulas-custom.adoc[].
-
 All formulas must be accurately configured for your {smr} installation to function correctly.
 If you are unsure of the correct formula configuration details, run the [command]``retail_branch_init`` command before you begin to create the recommended formula configuration.
 You can then manually edit the formulas as required.
 
+== {smr} Branch Server Formulas
+
+Branch servers are configured using formulas. Formulas can be configured using {susemgr} {webui}, and {susemgr} XMLRPC API.
+To fully configure {smr} environment, following formulas need to be enabled and configured for branch server:
+
+* Branch Network Formula, see xref:salt:formula-branchnetwork.adoc[]
+* Bind Formula, see xref:salt:formula-bind.adoc[]
+* Dhcpd Formula, see xref:salt:formula-dhcpd.adoc[]
+* PXE Formula, see xref:salt:formula-pxe.adoc[]
+* TFTP Formula, see xref:salt:formula-tftpd.adoc[]
+* Vsftp Formula, see xref:salt:formula-vsftpd.adoc[]
+
+Optionally, [systemitem]``Image Synchronization Formula`` can also be enabled and configured.
+For more information, see xref:salt:formula-imagesync.adoc[].
 
-.State and Formula Name Collisions
 [IMPORTANT]
 ====
+It is very easy to misconfigure formulas that may result in branch not working properly.
```

Comment:
```suggestion
Badly configured formulas can result in the branch server failing.
```

Reply:
Well, the server will be running, just something will not work as expected. For example if someone misconfigure bind formula, terminal will get the proper ip address, will properly start saltboot, but then can fail because of bad hostname resolution.

Reply:
How about this?